mappings/net/minecraft/data/SnbtProvider.mapping

```diff
@@ -19,10 +28,12 @@ CLASS net/minecraft/class_2463 net/minecraft/data/SnbtProvider
 		FIELD field_20538 name Ljava/lang/String;
 		FIELD field_20539 bytes [B
 		FIELD field_20540 sha1 Ljava/lang/String;
+		FIELD field_24616 jsonizedContent Ljava/lang/String;
```

Comment:
```suggestion
		FIELD field_24616 snbtContent Ljava/lang/String;
```

Reply:
I thought this was already applied? 🤔

Reply:
You only applied it to the parameter at first, I think.